vnext/local-cli/generator-windows/templates/cpp/proj/MyApp.vcxproj

```diff
@@ -70,6 +70,7 @@
     <WholeProgramOptimization>true</WholeProgramOptimization>
     <LinkIncremental>false</LinkIncremental>
   </PropertyGroup>
+  <%=XamlProps%>
```

Comment:
I don't love that we're adding more props here.

This feels like the time to make the breaking change of pulling more props/targets into the RNW package. We do that already for bundling IIRC, but with current structure we will keep on adding changes that users will need to manually update.

E.g. Something like a Microsoft.ReactNative.targets inside of the package where we can freely add to without changing the template. (Not sure if that name is in use right now).

Reply:
@NickGerleman this is something @jonthysell  is working on (and @dannyvv mentioned as well, and has a proposal out for).

Reply:
taking this up again now that Jon's change is in. After we talked about it I think this is still the right change - the app should be in complete control of its dependencies, we should just give it a good default to start with (2.x or 3.0).